serializer.rst

```diff
@@ -1472,6 +1472,255 @@ like:
+                <!-- ... -->
+
+                <!-- Disable autoconfigure to prevent this normalizer from automatically -->
+                <!-- being included in the default serializer -->
+                <service
+                    id="Symfony\Component\Serializer\Normalizer\CustomNormalizer"
+                    autoconfigure="false"
+                >
+                    <!-- Include this normalizer in a single serializer -->
+                    <tag name="serializer.normalizer" serializer="api_client1"/>
+
+                    <!-- Include this normalizer in multiple serializers -->
+                    <tag name="serializer.normalizer" serializer="api1"/>
+                    <tag name="serializer.normalizer" serializer="api2"/>
+
+                    <!-- Include this normalizer in all serializers (including the default one) -->
+                    <tag name="serializer.normalizer" serializer="*"/>
+                </service>
+            </services>
+        </container>
+
+    .. code-block:: php
+
+        // config/services.php
+        namespace Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+        use Symfony\Component\Serializer\Normalizer\CustomNormalizer;
+
+        return function(ContainerConfigurator $container) {
+            // ...
+
+            $services->set(CustomNormalizer::class)
+                // Prevent this normalizer from automatically being included in the default serializer
+                ->autoconfigure(false)
+
+                // Include this normalizer in a single serializer
+                ->tag('serializer.normalizer', ['serializer' => 'api_client1'])
+                // Include this normalizer in multiple serializers
+                ->tag('serializer.normalizer', ['serializer' => ['api_client1', 'api_client2']])
+                // Include this normalizer in all serializers (including the default one)
+                ->tag('serializer.normalizer', ['serializer' => '*'])
+            ;
+        };
+
+When the ``serializer`` attribute is not set, the service is registered with
+the default serializer.
+
+Each normalizer and encoder used in a named serializer is tagged with
+a ``serializer.normalizer.<name>`` or ``serializer.encoder.<name>`` tag,
+which can be used to list their priorities using the following command:
+
+.. code-block:: terminal
+
+    $ php bin/console debug:container --tag serializer.<normalizer|encoder>.<name>
+
+Additionally, you can exclude the default set of normalizers and encoders by
+setting the ``include_built_in_normalizers`` and ``include_built_in_encoders``
+options to ``false``:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/serializer.yaml
+        framework:
+            serializer:
+                named_serializers:
+                    api_client1:
+                        include_built_in_normalizers: false
+                        include_built_in_encoders: true
```

Comment:
The check fails because the default set of normalizers has been disabled, and no custom normalizer has been registered. But this is expected behavior.